docs/changes/io.fits/12544.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Add a ``mask_invalid`` option to ``Table.read`` to deactivate the masking of
```

Comment:
Maybe "... to allow deactivating ..." (since by default the option activates it)